web/src/search/backend.tsx

```diff
@@ -558,18 +554,26 @@ export function fetchRecentSearches(userId: GQL.ID, first: number): Observable<E
                 }
             }
         `,
-        variables: { userId, first: first ?? null },
+        variables: { userId, first: first ?? null, eventName },
     })
 
     return result.pipe(
         map(dataOrThrowErrors),
         map(
-            (data: RecentSearchesPanelDataResult): EventLogResult => {
+            (data: EventLogsDataResult): EventLogResult => {
                 if (!data.node) {
                     throw new Error('User not found')
                 }
-                return data.node.recentSearches
+                return data.node.eventLogs
             }
         )
     )
 }
+
+export function fetchRecentSearches(userId: GQL.ID, first: number): Observable<EventLogResult | null> {
+    return fetchEvents(userId, first, 'SearchResultsQueried')
+}
+
+export function fetchRecentFiles(userId: GQL.ID, first: number): Observable<EventLogResult | null> {
```

Comment:
```suggestion
export function fetchRecentFileViews(userId: GQL.ID, first: number): Observable<EventLogResult | null> {
```
Perhaps a better name for what this is fetching